plugins/inventory/scaleway.py

```diff
@@ -1,36 +1,38 @@
-# Copyright (c) 2017 Ansible Project
+# Copyright: (c) 2017 Ansible Project
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
 
 __metaclass__ = type
 
-DOCUMENTATION = '''
+DOCUMENTATION = r'''
     name: scaleway
     author:
       - Remy Leone (@sieben)
     short_description: Scaleway inventory source
     description:
-        - Get inventory hosts from Scaleway
+        - Get inventory hosts from Scaleway.
     options:
         plugin:
             description: token that ensures this is a source file for the 'scaleway' plugin.
```

Comment:
```suggestion
            description: Token that ensures this is a source file for the 'scaleway' plugin.
```
There's also the `variables` option's description to capitalize the first letter.